src/utils/transaction.ts

```diff
@@ -685,8 +685,7 @@ const transaction = {
       histTx.nc_method = (tx as NanoContract).method;
       histTx.nc_args = (tx as NanoContract).args.map(a => a.toString('hex')).join('');
       histTx.nc_pubkey = (tx as NanoContract).pubkey.toString('hex');
-      // XXX: is the nc_blueprint_id required?
-      // histTx.nc_blueprint_id
+      // Cannot fetch histTx.the nc_blueprint_id with the current data
```

Comment:
typo(non-blocking): Is the `the` word correct here?
```suggestion
      // Cannot fetch histTx.nc_blueprint_id with the current data
```